tests/unit/factories/canonical-data-factory-test.ts

```diff
@@ -0,0 +1,64 @@
+import { expect } from 'chai'
+
+import Factory from '../../../src/factories/canonical-data-factory';
+import { Feature } from '../../../src/factories/factory-types'
+import NamedNode from '../../../src/named-node'
+import Literal from '../../../src/literal'
+import DefaultGraph from '../../../src/default-graph'
+import Empty from '../../../src/empty'
+
+describe('data-factory', () => {
+  describe('DataFactory', () => {
+    describe('#supports', () => {
```

Comment:
The idea behind the test functions being called `it` is that you can make somewhat natural-sounding tests, so it'd probably be more descriptive to remove this `describe('#supports`, and instead use `it('supports collections'`, etc.

Reply:
Especially since it appears that some should be `it('does not support defaultGraphType'.

Reply:
Agreed, edited.